ConEmuWinForms/GlobalSuppressions.cs

```diff
@@ -0,0 +1,8 @@
+﻿// This file is used by Code Analysis to maintain SuppressMessage
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given
+// a specific target and scoped to a namespace, type, member, etc.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Usage", "VSTHRD110:Observe result of async calls", Justification = "Upsteam implementation", Scope = "member", Target = "~P:ConEmu.WinForms.ConEmuControl.IsStatusbarVisible")]

```

Comment:
"Upst**r**eam"
(Code suggestions are not enabled for this project.)

Reply:
I saw you made a comment locally but not what it was, was waiting for you to review.
Next time...